src/main/java/com/sprint/mission/discodeit/repository/jcf/JCFBinaryContentRepository.java

```diff
@@ -0,0 +1,57 @@
+package com.sprint.mission.discodeit.repository.jcf;
+
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import com.sprint.mission.discodeit.repository.BinaryContentRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public class JCFBinaryContentRepository implements BinaryContentRepository {
+    private final List<BinaryContent> data = new ArrayList<>();
+    @Override
+    public void delete(UUID id) {
+        data.removeIf(bc -> bc.getId().equals(id));
```

Comment:
```suggestion
        data.removeIf(binaryContent -> binaryContent.getId().equals(id));
```
이런 경우는 명확하게 변수명 표현하시는경우가 훨씬 직관적인 구현이 가능합니다.